site/proxy.md

```diff
@@ -40,13 +40,18 @@ interfaces. This can be adjusted with the `-H` argument, e.g.
 
     host1$ weave launch-proxy -H tcp://127.0.0.1:9999
 
-All docker commands can be run via the proxy, so it is safe to
-globally adjust your `DOCKER_HOST` to point at the proxy, e.g.
+All docker commands can be run via the proxy, so it is safe to use
+`weave proxy-env` to globally adjust your `DOCKER_HOST` to point at
+the proxy, e.g.
```

Comment:
"All docker commands can be run via the proxy, so it is safe to adjust your `DOCKER_HOST` to point at the proxy. Weave provides a convenient command for this:"